### PR TITLE
SW-1666 Viability testing database - version 1

### PR DESCRIPTION
**Pull request by karthikbtf**
- this version shows data as required/specified in designs
- ability to create a new test from database, triggers the new test modal in the parent component
- added new hook to view existing tests
- version 2 will be iterations on top of this if any are needed

viability-testing-database-desktop

viability-testing-database-tablet

viability-testing-database-mobile

### CODE REVIEW

File: src/components/accession2/viabilityTesting/TableCellRenderer.tsx [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+
+  if (column.key === 'viabilityPercent') {
+    const { viabilityPercent } = row;
+    if (!row.endDate || row.viabilityPercent === undefined) {
+      return (
+        <CellRenderer {...defaultProps} value={getValue(<Icon name='iconSynced' className={classes.syncIcon} />)} />
+      );
+    }
+
+    return <CellRenderer {...defaultProps} value={getValue(`${viabilityPercent}%`)} />;
```
Comment (constanzauanini) [inline src/components/accession2/viabilityTesting/TableCellRenderer.tsx:86]: This percentage shows bolder

**Review by constanzauanini — APPROVED** [review]
Looking good! One comment about style for viability percentage

File: src/components/accession2/viabilityTesting/TableCellRenderer.tsx (hunk 1 above)
Comment (karthikbtf) [reply src/components/accession2/viabilityTesting/TableCellRenderer.tsx:86]: in the mocks or here in the vercel preview?

Comment (constanzauanini) [reply src/components/accession2/viabilityTesting/TableCellRenderer.tsx:86]: In the mocks looks bolder than in the preview (for me)

Comment (karthikbtf) [reply src/components/accession2/viabilityTesting/TableCellRenderer.tsx:86]: > In the mocks looks bolder than in the preview (for me)

I see it now, thanks 👍

Comment (karthikbtf) [reply src/components/accession2/viabilityTesting/TableCellRenderer.tsx:86]: mock has fontWeight: 500, will update

**Comment by karthikbtf** [comment]
Fixed
Terraware App 2022-09-20 07-45-25

**Comment by constanzauanini** [comment]
> Fixed Terraware App 2022-09-20 07-45-25

Perfect. Thank you!!

**Review by constanzauanini — APPROVED** [review]

**Pull request merged**